hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemCreate.java

```diff
@@ -830,6 +830,13 @@ public void testCreateFileOverwrite(boolean enableConditionalCreateOverwrite)
         (AzureBlobFileSystem) FileSystem.newInstance(currentFs.getUri(),
             config);
 
+    int validatePathOrSubPathDoesNotExistConn = 0;
+    if (fs.getAbfsStore().getPrefixMode() == PrefixMode.BLOB) {
+      validatePathOrSubPathDoesNotExistConn = 2;
```

Comment:
rename validatePathOrSubPathDoesNotExistConn to whenBlob_CheckIfPathIsDirectory

Add below comments
// When blob extra call to check on path status is done:
// Connection 1: List on the path
// Connection 2: getproperties on the path
// if parent is not root, then mkdir, but the connections for these wont happen for this test run.